bindings/c/test/mako/mako.c

```diff
@@ -871,6 +954,14 @@ int worker_process_main(mako_args_t *args, int worker_id, mako_shmhdr_t *shm) {
           "ERROR: fdb_network_set_option(FDB_NET_OPTION_TRACE_ENABLE): %s\n",
           fdb_get_error(err));
     }
+    err = fdb_network_set_option(FDB_NET_OPTION_TRACE_FORMAT,
```

Comment:
Is this something that could be made into an argument? It's a bit awkward to have different choices for default trace format in the codebase.

Reply:
Yeah, I was thinking the same, but just being lazy.  😁
Just parameterized it in the most recent commit.